stdlib/REPL/src/LineEdit.jl

```diff
@@ -1548,7 +1548,7 @@ function normalize_keys(keymap::Union{Dict{Char,Any},AnyDict})
     return ret
 end
 
-function add_nested_key!(keymap::Dict, key::Union{String, Char}, value; override = false)
+function add_nested_key!(keymap::Dict, key::Union{String, Char}, value; override::Bool = false)
```

Comment:
```suggestion
function add_nested_key!(keymap::Dict{Char, Any}, key::Union{String, Char}, value; override::Bool = false)
```

alternatively?

Reply:
Will that also be true for the line 
```julia
keymap[c] = value
```
below? I'm not that familiar with this code, that's why I'm asking.

Reply:
Not sure what you mean but I think `keymap` is always  a `Dict{Char, Any}` indeed.

Reply:
Thanks. I applied your suggestion in 6f4b2606ebf79f6bc203d8de4ad3e7b9d60e8dd3.